### PR TITLE
OPS-631: Upgrade to the latest version of python

### PR DESCRIPTION
**Pull request by feanil**

### CODE REVIEW

File: playbooks/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -31,11 +31,16 @@
...
+- name: Add python 2.7 apt repository
+  apt_repository: repo="{{ common_python_ppa }}"
+
 - name: Install role-independent useful system packages
   # do this before log dir setup; rsyslog package guarantees syslog user present
   apt: >
-    pkg={{','.join(common_debian_pkgs)}} install_recommends=yes
+    pkg="{{ item }}" install_recommends=yes
     state=present update_cache=yes
+  with_items: common_debian_pkgs
```
Comment (maxrothman) [inline playbooks/roles/common/tasks/main.yml:36]: Why this change?

Comment (feanil) [inline playbooks/roles/common/tasks/main.yml:36]: The join was an optimization to install all items in one transaction. That optimization is no longer required is my understanding.

**Comment by maxrothman** [comment]
:+1:

**Comment by jibsheet** [comment]
:+1:
